packages/core/src/test/amazonqFeatureDev/prepareRepoData.test.ts

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import assert from 'assert'
+import { WorkspaceFolder } from 'vscode'
+import { performanceTest } from '../../shared/performance/performance'
+import { createTestWorkspace } from '../testUtil'
+import { prepareRepoData, TelemetryHelper } from '../../amazonqFeatureDev'
+import { AmazonqCreateUpload, getRandomString, Metric } from '../../shared'
+
+type resultType = {
+    zipFileBuffer: Buffer
+    zipFileChecksum: string
+}
+
+describe('prepareRepoDataPerformanceTest', function () {
```

Comment:
I'd probably just have a top level describe thats prepareRepoData and then a second describe that has performance test in it, that way it can easily seperate perf tests from non perf tests if anyone wants to add it

Reply:
There are also some existing correctness tests in `test/unit/amazonqFeatureDev/util/files.test.ts`. I kept the performance tests in core since thats where the `startSecurityScan` is (src/test/codewhisperer/commands/startSecurityScan.test.ts).